src/main/java/com/codecool/virtualstylist/wardrobe/ClothForCreationDTO.java

```diff
@@ -0,0 +1,132 @@
+package com.codecool.virtualstylist.wardrobe;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class ClothForCreationDTO {
+
+    @JsonProperty("imagePath")
+    private String imagePath;
+
+    @JsonProperty("clothType")
+    private String clothType;
+
+    @JsonProperty("color")
+    private String color;
+
+    @JsonProperty("code")
+    private String code;
+
+    @JsonProperty("brand")
+    private String brand;
+
+    @JsonProperty("style")
+    private String style;
+
+    @JsonProperty("tag")
+    private String tag;
+
+    @JsonProperty("size")
+    private String size;
+
+    @JsonProperty("hasPattern")
+    private boolean hasPattern;
+
+    @JsonProperty("shopLink")
+    private String shopLink;
+
```

Comment:
Shouldn't this object aggregate user?

Reply:
user will be found in WardrobeService and set on actual Cloth object, we do not receive information about user from front-end on this endpoint